Scenes/World/Layer.gd

```diff
@@ -4,7 +4,7 @@ var TREE_TEXTURES = [
 	load("res://Objects/Background/Trees/tree1.png"),
 	load("res://Objects/Background/Trees/tree2.png"),
 	load("res://Objects/Background/Trees/tree3.png"),
-	load("res://Objects/Background/Trees/tree4.png"),
+	#load("res://Objects/Background/Trees/tree4.png"),
```

Comment:
Why the tree4 is worse than the rest?

Reply:
We was talking about this in the Monday's meeting and we decited to remove that tree, becouse it's not matches other. I think I should remove that line. Do I?

Reply:
Yes, please